.github/dependabot.yml

```diff
@@ -5,23 +5,38 @@ updates:
     open-pull-requests-limit: 999
     rebase-strategy: disabled
     schedule:
-      interval: daily
+      interval: weekly
       time: "00:00"
       timezone: Etc/GMT
     labels:
       - area/dependencies
       - area/ci
+    groups:
```

Comment:
If I'm reading this correctly, it will group security updates. Not sure if this is a good idea, as we want to see them separately. Can we start with grouping version updates only first?